pkg/utils/rfctime/rfc3339.go

```diff
@@ -18,6 +18,31 @@ const RFC3339DateTimeFormat string = "2006-01-02T15:04:05.999-07:00"
 // Use it to parse RFC3339 date-time expression.
 const RFC3339DateTimeFormatZ string = time.RFC3339Nano
 
+// The following format is used to parse the abbreviated form of RFC3339 date-time.
+const (
+	RFC3339DateNano       = "2006-01-02T15:04:05.999999999"
+	RFC3339DateNanoNoDeL  = "2006-01-02 15:04:05.999999999"
+	RFC3339DateNanoZNoDeL = "2006-01-02 15:04:05.999999999Z07:00"
+
+	RFC3339DateSec       = "2006-01-02T15:04:05"
+	RFC3339DateSecZ      = "2006-01-02T15:04:05Z07:00"
+	RFC3339DateSecNoDeL  = "2006-01-02 15:04:05"
```

Comment:
```suggestion
	RFC3339DateSecSpace  = "2006-01-02 15:04:05"
```

Trying to keep concise is good. Choicing clear-cut name makes it better.

In this case, NoDeL(imiter? I suppose) does not match with the format.
**There are delimiter** between the date and the time, **space** .

Both `NoDeL` and `Space` are 5 characters. `Space` has no omitted chalacters. This is better.